template/web/src/components/layout/header/AccountDropdown.tsx

```diff
@@ -4,6 +4,10 @@ import { clsx } from 'clsx';
 import { useAccount } from 'wagmi';
 import { AccountInfoPanel } from './AccountInfoPanel';
 
+const DropdownMenuContentStyle = {
+  marginTop: '-22px',
+};
+
```

Comment:
Yes I hate this too. Can't figure out why radix-ui bumps down the Navbar so far down.